auth-web/src/components/auth/staff/StaffActiveAccountsTable.vue

```diff
@@ -227,11 +233,12 @@ export default class StaffActiveAccountsTable extends Vue {
   opacity: 0;
   overflow: hidden;
   max-height: 0;
-  transition: all ease-out 0.5s;
-  &.active {
-    opacity: 1;
-    max-height: 4rem;
-  }
+  transition: all ease-out 0.25s;
```

Comment:
Animation transition was not consistent with existing patterns (see Transactions filters)